qemu/tests/nic_opt.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from virttest import utils_net
+from virttest import utils_test
+from virttest import error_context
+from avocado.utils import cpu
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    NIC option check test.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def test_netperf():
+        """
+        Netperf stress test for nic option.
+        """
+        netperf_client = params.get("netperf_client", "main_vm")
+        netperf_server = params.get("netperf_server", "localhost")
+        sub_type = params.get("sub_type", "netperf_stress")
+        params["netperf_client"] = netperf_client
+        params["netperf_server"] = netperf_server
```

Comment:
I believe LINE 26 and 27 are redundant.

Reply:
ok, I will test it. 
if test result is normally.I will delete it.

Reply:
I have deleted it,please help review again.Thanks  a lot.